cmd/machine-config-daemon/start.go

```diff
@@ -202,6 +204,27 @@ func runStartCmd(cmd *cobra.Command, args []string) {
 	stopCh := make(chan struct{})
 	defer close(stopCh)
 
+	if startOpts.desiredConfigMap != "" {
```

Comment:
Only the presence of the configmap indicates hypershift? there's not an annotation or something unambiguous?

Reply:
This is a flag in the daemon which essentially indicates "hypershift-mode". I can instead name it like hypershiftDesiredConfigMap to add clarity

By annotation do you mean pod annotation? Nodes are not yet populated and I would need to determine what pod mode I am running in, which is the flag.

Reply:
renaming wfm